It seems there is an unrelated error in a test of the module mrp_multi_level, I have to check that.

```
2024-08-26 06:43:58,824 433 INFO odoo odoo.addons.mrp_multi_level.tests.test_mrp_multi_level: ====================================================================== 
2024-08-26 06:43:58,824 433 ERROR odoo odoo.addons.mrp_multi_level.tests.test_mrp_multi_level: FAIL: TestMrpMultiLevel.test_04_mrp_multi_level
Traceback (most recent call last):
  File "/__w/manufacture/manufacture/mrp_multi_level/tests/test_mrp_multi_level.py", line 120, in test_04_mrp_multi_level
    self.assertEqual(len(sf_1_line_2), 1)
AssertionError: 0 != 1
 
```

--- a/account_move_line_mrp_info/views/account_move_line_view.xml
+++ b/account_move_line_mrp_info/views/account_move_line_view.xml
@@ -12,4 +12,16 @@
         </field>
     </record>
 
+    <record id="view_account_move_line_filter" model="ir.ui.view">
+        <field name="name">account.move.line.search</field>
+        <field name="model">account.move.line</field>
+        <field name="inherit_id" ref="account.view_account_move_line_filter" />
+        <field name="arch" type="xml">
+            <field name="partner_id" position="after">
+                <field name="mrp_production_id" />
+                <field name="unbuild_id" />
+            </field>
+        </field>
+    </record>
+
 </odoo>

